Welcome Page Redesign

skipcheru opened the pull request:
#### This PR introduces the new design of the welcome page based on the mock-ups.

####  Screenshot.
![Screenshot_20190713-192943](https://user-images.githubusercontent.com/4677105/61174154-21658900-a5a5-11e9-950f-4c32f0f13486.png)

noahpierau commented:
Please get the logo from Decred.org/brand 

Spelling it as "deCRED" looks very bad!

C-ollins commented on app/src/main/java/com/dcrandroid/activities/SetupWalletActivity.java:
```diff
@@ -62,6 +61,16 @@ public void onClick(View view) {
         });
-    }
 
+        tvViewSource.setMovementMethod(LinkMovementMethod.getInstance());
```
This isn't needed since the text content doesn't contain a link.

denyszayets commented:
Not sure where this layout is coming from but it we need to use brand art files. Logo etc

DENYS ZAYETS UI/UX Graphic Designer
(904) 613-5591 denyszayets.com [http://denyszayets.com/] & @denyszayets [https://twitter.com/denyszayets]
On Sat, Jul 13, 2019 at 12:35, Sam Cheruiyot <notifications@github.com> wrote:
THIS PR INTRODUCES THE NEW DESIGN OF THE WELCOME PAGE BASED ON THE MOCK-UPS.
SCREENSHOT.
[https://user-images.githubusercontent.com/4677105/61174154-21658900-a5a5-11e9-950f-4c32f0f13486.png]




YOU CAN VIEW, COMMENT ON, OR MERGE THIS PULL REQUEST ONLINE AT:
https://github.com/decred/dcrandroid/pull/395 [https://github.com/decred/dcrandroid/pull/395]

COMMIT SUMMARY
 * initial commit for arrow, refresh and add icons
 * Refactor: change view to new design
 * Refactor: update views and create wallet click listener
 * Update: add welcome page strings

FILE CHANGES
 * M app/src/main/java/com/dcrandroid/activities/SetupWalletActivity.java [https://github.com/decred/dcrandroid/pull/395/files#diff-0] (31)
 * A app/src/main/res/drawable-anydpi/ic_add_white.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-1] (10)
 * A app/src/main/res/drawable-anydpi/ic_arrow_forward_blue.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-2] (10)
 * A app/src/main/res/drawable-anydpi/ic_refresh_white.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-3] (10)
 * A app/src/main/res/drawable-anydpi/ic_restore_white.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-4] (10)
 * A app/src/main/res/drawable-hdpi/ic_add_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-5] (0)
 * A app/src/main/res/drawable-hdpi/ic_arrow_forward_blue.png [https://github.com/decred/dcrandroid/pull/395/files#diff-6] (0)
 * A app/src/main/res/drawable-hdpi/ic_refresh_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-7] (0)
 * A app/src/main/res/drawable-hdpi/ic_restore_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-8] (0)
 * A app/src/main/res/drawable-mdpi/bg_blue_corner_8dp.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-9] (14)
 * A app/src/main/res/drawable-mdpi/bg_turquoise_corner_8dp.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-10] (14)
 * A app/src/main/res/drawable-mdpi/ic_add_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-11] (0)
 * A app/src/main/res/drawable-mdpi/ic_arrow_forward_blue.png [https://github.com/decred/dcrandroid/pull/395/files#diff-12] (0)
 * A app/src/main/res/drawable-mdpi/ic_refresh_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-13] (0)
 * A app/src/main/res/drawable-mdpi/ic_restore_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-14] (0)
 * A app/src/main/res/drawable-xhdpi/ic_add_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-15] (0)
 * A app/src/main/res/drawable-xhdpi/ic_arrow_forward_blue.png [https://github.com/decred/dcrandroid/pull/395/files#diff-16] (0)
 * A app/src/main/res/drawable-xhdpi/ic_refresh_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-17] (0)
 * A app/src/main/res/drawable-xhdpi/ic_restore_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-18] (0)
 * A app/src/main/res/drawable-xxhdpi/ic_add_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-19] (0)
 * A app/src/main/res/drawable-xxhdpi/ic_arrow_forward_blue.png [https://github.com/decred/dcrandroid/pull/395/files#diff-20] (0)
 * A app/src/main/res/drawable-xxhdpi/ic_refresh_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-21] (0)
 * A app/src/main/res/drawable-xxhdpi/ic_restore_white.png [https://github.com/decred/dcrandroid/pull/395/files#diff-22] (0)
 * M app/src/main/res/layout/activity_setup_page.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-23] (281)
 * M app/src/main/res/values/strings.xml [https://github.com/decred/dcrandroid/pull/395/files#diff-24] (9)

PATCH LINKS:
 * https://github.com/decred/dcrandroid/pull/395.patch [https://github.com/decred/dcrandroid/pull/395.patch]
 * https://github.com/decred/dcrandroid/pull/395.diff [https://github.com/decred/dcrandroid/pull/395.diff]

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub [https://github.com/decred/dcrandroid/pull/395?email_source=notifications&email_token=AIBNI42SW3P5BUJCWJIBQD3P7H745A5CNFSM4IC7CUGKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4G7BFEGA] , or mute the thread [https://github.com/notifications/unsubscribe-auth/AIBNI46ZPM4DSYRFREBEL3TP7H745ANCNFSM4IC7CUGA] .

skipcheru commented:
> Please get the logo from Decred.org/brand
> 
> Spelling it as "deCRED" looks very bad!

Thanks for the catch. I've updated the logo and fonts as per Collins comments also. 
This is how it looks now.

![Screenshot_20190715-201235](https://user-images.githubusercontent.com/4677105/61235242-bb603980-a73d-11e9-84c7-4a95fc959505.png)

denyszayets commented:
Do you know who made these visual changes/revisions?

On Mon, Jul 15, 2019 at 13:20, Sam Cheruiyot <notifications@github.com> wrote:
Please get the logo from Decred.org/brand

Spelling it as "deCRED" looks very bad!

Thanks for the catch. I've updated the logo and fonts as per Collins comments also.
This is how it looks now.

[https://user-images.githubusercontent.com/4677105/61235242-bb603980-a73d-11e9-84c7-4a95fc959505.png]

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub [https://github.com/decred/dcrandroid/pull/395?email_source=notifications&email_token=AIBNI4YF3TBHZW4QIHWLPKLP7SWVXA5CNFSM4IC7CUGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZ6L7CI#issuecomment-511491977] , or mute the thread [https://github.com/notifications/unsubscribe-auth/AIBNI42I4FYOYMRWVJAZNPDP7SWVXANCNFSM4IC7CUGA] .

morphymore commented:
Hi @skipcheru & @C-ollins,

Please follow the design specs from [Gallery](https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScQvwJU5rom_irDf2PPnvm8A).
You should be able to check the widths, heights, spaces, and which icon/image to use under **Inspect** mode.

<img width="625" alt="Screen Shot 2019-07-16 at 11 34 37" src="https://user-images.githubusercontent.com/13583235/61264733-26525480-a7c0-11e9-8c41-9eafb225d5b9.png">

For example, the Decred logo should be **ic_logo_primary**, and you can find the corresponding file in the [icon folder](https://drive.google.com/drive/folders/1ZjbFq0pJbqZI2mOE25OKDdkDPf9d2ZUh?usp=sharing). All exported icons/images can be found there (exporting in progress).

<img width="371" alt="Screen Shot 2019-07-16 at 11 50 56" src="https://user-images.githubusercontent.com/13583235/61264972-00797f80-a7c1-11e9-988b-a40efc164ec1.png">
<img width="362" alt="Screen Shot 2019-07-16 at 11 59 11" src="https://user-images.githubusercontent.com/13583235/61265012-29017980-a7c1-11e9-87eb-1bafbbaf038d.png">

Also, there were and will be some minor changes to the designs (e.g. the github link is now removed). Please refresh regularly to see the latest iteration. Keep up the good work!

skipcheru commented:
Hi @morphymore and @denyszayets  thanks for the feedback I've tried to replicate the design as much as I can now and also used the icons you shared.
I also wanted to know if the designs are still changing or the current version final. Thanks for sharing the icons.

This is how it looks now.

![Screenshot_20190716-214431](https://user-images.githubusercontent.com/4677105/61320931-334c6380-a813-11e9-8443-baada09f4228.png)

denyszayets commented:
Happy to hear you guys are working on it ;) Please let me know if you need anything from original mockups

DENYS ZAYETS UI/UX Graphic Designer
(904) 613-5591 denyszayets.com [http://denyszayets.com/] & @denyszayets [https://twitter.com/denyszayets]
On Tue, Jul 16, 2019 at 14:49, Sam Cheruiyot <notifications@github.com> wrote:
Hi @morphymore [https://github.com/morphymore] and @denyszayets [https://github.com/denyszayets] thanks for the feedback I've tried to replicate the design as much as I can now and also used the icons you shared.
I also wanted to know if the designs are still changing or the current version final. Thanks for sharing the icons.

This is how it looks now.

[https://user-images.githubusercontent.com/4677105/61320931-334c6380-a813-11e9-8443-baada09f4228.png]

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub [https://github.com/decred/dcrandroid/pull/395?email_source=notifications&email_token=AIBNI46MZZI2UNDCJVH3X2DP7YJ3TA5CNFSM4IC7CUGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2BZKYQ#issuecomment-511939938] , or mute the thread [https://github.com/notifications/unsubscribe-auth/AIBNI46OPTJEJHDQWRTPYQDP7YJ3TANCNFSM4IC7CUGA] .

morphymore commented:
Nice work @skipcheru 

Regarding this welcome screen, it is final.

And a few things to check again:
1. Button text font size (a bit small, it should be 16sp)
2. Button icon paddings (inconsistent, top/ left/ bottom paddings should all be 16dp)
3. Button drop shadow

---

Hello @denyszayets, which mockup are you referring to?

skipcheru commented:
Hi @morphymore these are the Issues I fixed:
 - Button text font size and shadow
 - Icon padding

For the button shadow, I added elevation of 1dp. hope it matches what its on the design doc.

This is the look now:

![Screenshot_20190719-153255](https://user-images.githubusercontent.com/4677105/61535750-b2bc7b80-aa3b-11e9-9ba5-05a4a0f33642.png)

morphymore commented:
Nice @skipcheru let's check on some details:

1. Button text: Is the font size 16sp already? It is certainly larger than the previous one, but still doesn't look consistent with the designed screen. (If it's indeed 16sp in your code, we need to figure out what makes the outcomes inconsistent)

<img width="1207" alt="Screen Shot 2019-07-21 at 13 51 47" src="https://user-images.githubusercontent.com/13583235/61587560-00adbc80-abbf-11e9-9e76-ff0a5759255e.png">

2. Button shadows: The shadow effect on buttons are of 1dp elevation with a **blur attribute of 4** (as you can check under the Shape column).

<img width="362" alt="Screen Shot 2019-07-21 at 13 44 49" src="https://user-images.githubusercontent.com/13583235/61587492-d0195300-abbd-11e9-92c9-0512dd00e6ae.png">

Also I've noticed everything in your screen are about 1.125x smaller (320 to 360) to the design screen, as you can see in the image above. Can you think of anything that might be causing this?

C-ollins commented:
Moved to #400
